<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DoxygenExport</class>
 <widget class="QDialog" name="DoxygenExport">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>552</width>
    <height>207</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>DoxygenExport</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Featerues</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="checkBoxCreateFormPage">
        <property name="text">
         <string>Create Form Page</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxCreateResourcePage">
        <property name="text">
         <string>Create Resource Page</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Files</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="3">
         <widget class="QToolButton" name="btnSelectImageFolder">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Image Folder</string>
          </property>
          <property name="buddy">
           <cstring>editImageFolder</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLineEdit" name="editImageFolder"/>
        </item>
        <item row="0" column="2">
         <widget class="QLineEdit" name="editDoxFileName"/>
        </item>
        <item row="0" column="3">
         <widget class="QToolButton" name="btnSelectDoxyFileName">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Doxy Filename</string>
          </property>
          <property name="buddy">
           <cstring>editDoxFileName</cstring>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DoxygenExport</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>424</x>
     <y>184</y>
    </hint>
    <hint type="destinationlabel">
     <x>0</x>
     <y>171</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DoxygenExport</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>153</x>
     <y>180</y>
    </hint>
    <hint type="destinationlabel">
     <x>5</x>
     <y>157</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
